shared/src/libs/packages/http/libs/enums/http-header.enum.ts

```diff
@@ -1,6 +1,7 @@
 const HttpHeader = {
   CONTENT_TYPE: 'content-type',
   AUTHORIZATION: 'authorization',
+  SHARED_ARTICLE_TOKEN: 'shared-article-token',
```

Comment:
Move it to separate enum called CustomHttpHeader, cause it's not a native headers, also custom headers should start from `x-` (eg `x-shared-article-token`)